pkg/sql/opt/exec/execbuilder/testdata/show_trace_nonmetamorphic

```diff
@@ -287,35 +274,27 @@ CREATE TABLE t (a INT PRIMARY KEY)
 statement ok
 SET tracing = on; SELECT * FROM t; SET tracing = off
 
-# Temporarily disabled flaky test (#58202).
-# query III colnames
-# SELECT DISTINCT node_id, store_id, replica_id
-#   FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
-# ----
-# node_id  store_id  replica_id
-# 1        1         26
-# 1        1         6
-# 1        1         29
-#
-# subtest system_table_lookup
-#
-# # We use AOST to bypass the table cache.
-# statement ok
-# SET tracing = on,kv; SELECT * FROM system.eventlog AS OF SYSTEM TIME '-1us'; SET tracing = off
-#
-# query TT
-# SELECT operation, regexp_replace(regexp_replace(message, 'job_id:[1-9]\d*', 'job_id:...', 'g')), 'drop_time:\d+', 'drop_time:...', 'g') as message
-#   FROM [SHOW KV TRACE FOR SESSION]
-# WHERE (message LIKE 'querying next range%' OR message LIKE '%batch%')
-#   AND message NOT LIKE '%SystemConfigSpan%'
-#   AND message NOT LIKE '%PushTxn%'
-# ----
-# dist sender send  querying next range at /Table/3/1/12/2/1
-# dist sender send  r6: sending batch 1 Get to (n1,s1):1
-# dist sender send  querying next range at /Table/3/1/1/2/1
-# dist sender send  r6: sending batch 1 Get to (n1,s1):1
-# dist sender send  querying next range at /Table/12/1
-# dist sender send  r8: sending batch 1 Scan to (n1,s1):1
+# This test shows some variation across runs so use statement okay instead
+# of baking in flakey values.
+statement ok
+SELECT DISTINCT node_id, store_id, replica_id
+  FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
+
+subtest system_table_lookup
+
+# We use AOST to bypass the table cache.
+statement ok
+SET tracing = on,kv; SELECT * FROM system.eventlog AS OF SYSTEM TIME '-1us'; SET tracing = off
+
+query TT
+SELECT operation, regexp_replace(regexp_replace(message, 'job_id:[1-9]\d*', 'job_id:...', 'g'), 'drop_time:\d+', 'drop_time:...', 'g') as message
```

Comment:
What's with all of the regexp_replace that then doesn't seem to apply to anything in the trace? Kinda feels like this has rotted beyond recognition & purpose.